modules/ecs_policy/instance_policy.tf

```diff
@@ -0,0 +1,79 @@
+data "aws_iam_policy_document" "container_perms" {
+  statement {
+    actions = [
+      "ssm:GetParameters",
+    ]
+
+    resources = [
+      "arn:aws:ssm:ap-southeast-2:538673716275:parameter*",
```

Comment:
Can we use https://www.terraform.io/docs/providers/aws/d/caller_identity.html to dynamically get the account Id and ensure it works in any AWS account?